build_tools/aws-sdk-code-generator/spec/protocol-tests/input/smithy-rpc-v2-cbor.json

```diff
@@ -35,6 +35,7 @@
                     "uri": "/service/RpcV2Protocol/operation/EmptyInputOutput",
                     "body": "v/8=",
                     "headers": {
+                        "Accept": "application/cbor",
```

Comment:
Since these will probably be revereted in release (until new ones flow through).  Is there a risk of protocol tests failing with the OLD tests but this code?

Reply:
I wasn't planning on merging until those tests have arrived and produce new diff. I was just checking if they would pass